.storybook/es-dev-server.config.js

```diff
@@ -0,0 +1,21 @@
+const cjsTransformer = require('es-dev-commonjs-transformer');
+
+module.exports = {
+    responseTransformers: [
+        cjsTransformer(
```

Comment:
Is this because the new libraries from the React team at CJS? Or is there something else that hasn't updated to ES modules here?

Reply:
There's a deep dependency on a library [classnames](https://github.com/JedWatson/classnames) that the react packages depend upon that appears to need the CJS treatment.